src/tools/pkcs11-tool.c

```diff
@@ -4509,21 +4509,32 @@ static CK_RV write_object(CK_SESSION_HANDLE session)
 			n_privkey_attr++;
 		}
 #if !defined(OPENSSL_NO_EC)
-#ifdef EVP_PKEY_ED448
-		else if ((pk_type == EVP_PKEY_EC) || (pk_type == EVP_PKEY_ED25519) || (pk_type == EVP_PKEY_ED448)) {
-#else
-		else if ((pk_type == EVP_PKEY_EC) || (pk_type == EVP_PKEY_ED25519)) {
+/*  aappeas the precompiler */
```

Comment:
typo

Reply:
Outdated, "aappeas the precompiler" is no longer in the code.
There was a forced push which does include this line before your comment.